devops/openshift/workers-deploy.yml

```diff
@@ -137,6 +137,7 @@ parameters:
   - name: BUILD_TAG
     value: "0"
   - name: WORKERS_SECRET_NAME
+    value: sims-api-creds
```

Comment:
I am not following this default value. Do we need it?

Reply:
Just following the steps follwed in api-deploy.yml, also the name was not self understanding what it is, the value just gives an idea what this secret name is.